### ISSUE TITLE
Post-processing search and replace for tech-specific paths

### ISSUE THREAD

**Issue by phbernard** [RealFaviconGenerator/rfg-api#6]
Usually, icons path are something like `/path/to/icons`. But various technologies can make this quite different: `<%= asset_path ... %>`, `<?php icons_path() %>`, etc.

RFG API might handle this, but that could cause more harm than help. For example, a Rails-dedicated `manifest.json` file should be renamed to `manifest.json.erb`. But this is clearly not the responsibility of the API and the Ruby-minded client may miss the point.

Better let the client do everything, as it is done in https://github.com/RealFaviconGenerator/rails_real_favicon/blob/master/lib/generators/favicon_generator.rb

This feature could be useful to other technologies. Thus it would be good to have a help in rfg-api.

**Comment by markhalliwell** [comment]
Yeah, the API itself shouldn't handle this IMO. It should be left to whatever technology consumes the API, as I have demonstrated in https://github.com/RealFaviconGenerator/grunt-real-favicon/pull/34.
